tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -256,7 +278,7 @@ struct bpf_map_def SEC("maps") _name = { \
 
 /*=============================== INTERNAL STRUCTS ===========================*/
 
-typedef struct context {
+typedef struct internal_proc_context {
```

Comment:
Any reason for changing this name?
It is not only the proc context that is in this struct

Reply:
There's a name collision with the `enum context` in vmlinux.h. I'll change it to 'internal_context'.

Reply:
So maybe "event_context"?